fdbclient/include/fdbclient/StorageServerInterface.h

```diff
@@ -709,18 +710,24 @@ struct WaitMetricsRequest {
 	// Waits for any of the given minimum or maximum metrics to be exceeded, and then returns the current values
 	// Send a reversed range for min, max to receive an immediate report
 	constexpr static FileIdentifier file_identifier = 1795961;
+	TenantInfo tenantInfo;
```

Comment:
Can we make this `Optional<TenantInfo>` for clusters that don't use the tenant feature?

Reply:
Thanks for the explanation. Since it's optional, my point is that making it `Optional<TenantInfo>` type can save on serialization cost and is more explicit to indicate the optionality.

Reply:
That makes sense. I've updated the field.